examples/nodejs-ts-app/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "demo app",
   "main": "app.js",
   "scripts": {
-    "build": "opa build -t wasm -e 'example/hello' ./example.rego && tar -xzf ./bundle.tar.gz /policy.wasm",
+    "build": "opa build --capabilities ../_shared/capabilities-v0.27.1.json -t wasm -e 'example/hello' ./example.rego && tar -xzf ./bundle.tar.gz /policy.wasm",
```

Comment:
Sorry for the hassle, [I've just bumped our CI to use opa 0.28.0](https://github.com/open-policy-agent/npm-opa-wasm/pull/51) -- it won't need this then.